src/main/scala/org/allenai/deep_qa/experiments/datasets/Science.scala

```diff
@@ -60,17 +75,55 @@ object ScienceDatasets {
     omnibusMtfGradeEightDevQuestionsWithBuscBackground
   ), "/efs/data/dlfa/processed/omnibus_4_8_dev/")
 
+  /////////////////////////////////////////////////////////////////////
+  // Diagram Multiple True False
+  /////////////////////////////////////////////////////////////////////
+
   val diagramMtfQuestionsWithBuscBackground: JValue = makeBackgroundDataset(
     ScienceFiles.ai2diagramSentences_multipleTrueFalse_appendAnswer,
     "plain sentence",
     ScienceCorpora.buscElasticSearchIndex(10)
   )
 
+  /////////////////////////////////////////////////////////////////////
+  // OpenQA Multiple True False
+  /////////////////////////////////////////////////////////////////////
+
   val openQaMtfQuestionsWithBuscBackground: JValue = makeBackgroundDataset(
     ScienceFiles.openQaAnimalAndGeneralScience_multipleTrueFalse_appendAnswer,
     "plain sentence",
     ScienceCorpora.buscElasticSearchIndex(10))
 
+  /////////////////////////////////////////////////////////////////////
+  // Omnibus QA
+  /////////////////////////////////////////////////////////////////////
+
+  /*
+   * Fourth Grade BUSC Background Files
+   */
+
+  val omnibusQaGradeFourTrainBuscBackgroundFile: JValue = SciQDatasets.makePassageBackgroundFile(
+    ScienceFiles.omnibusGradeFourTrainSentences_questionAndAnswer,
+    "question and answer",
+    ScienceCorpora.buscElasticSearchIndex(3)
+  )
+
+  val omnibusQaGradeFourDevBuscBackgroundFile: JValue = SciQDatasets.makePassageBackgroundFile(
```

Comment:
I had `ScienceDatasets` and `ScienceFiles` as two separate classes, and these would technically go under `ScienceFiles`.  I'm not thrilled by that organization, though; if you think there's a better way to organize all of this, feel free to make it better.